app/Views/CoaBooking/CaseSearch.cshtml

```diff
@@ -231,18 +231,18 @@
                                 }
 
                                 <div id="AppealAdditionalQs" class="row search-info-row"
-                                    style="display: @((Model.CaseType == CoaCaseType.Criminal || (Model.FactumFiled ?? false)) ? "block" : "none")">
+                                    style="display: @(Model.CaseType == CoaCaseType.Criminal || Model.FactumFiled is true ? "block" : "none")">
                                     <div class="col-md-6">
                                         <label>How long will you require for your hearing?</label>
                                         <div id="Appeal_IsFullDay" class="btn-radio-group">
                                             <label
-                                                class="btn btn-radio btn-radio--secondary btn-radio--day @GetValidationClass("IsFullDay") @((Model.IsFullDay ?? true) ? "" : "active") @(showingDates ? "disabled" : "")">
+                                                class="btn btn-radio btn-radio--secondary btn-radio--day @GetValidationClass("IsFullDay") @(Model.IsFullDay is false ? "active" : "") @(showingDates ? "disabled" : "")">
```

Comment:
👀 Much clearer now, I had to read the previous logic a couple of times to get what it was doing 👍